### PR TITLE
Upgrade to Camel 3.8.0

### PR DESCRIPTION
**Pull request by abel-connexta**
#### What does this PR do?
Upgrades to Camel 3.8.0 to continue receiving security updates. Also fixed issue for Admin Console, when creating a "WFS v1.1.0 Federated Source" configuration, you are left with loading spinning wheel icon and an json parsing exception in console.  

#### Who is reviewing it? 

@kentmorrissey
@shaundmorris
@zta6
@SmithJosh
#### Select relevant component teams: 
@codice/security
@codice/io

#### How should this be tested?

Test the http proxy and all the camel routes

###### **HttpProxy:**
F.Y.I - Also you will probably get a 403 response, that is ok, because that's the old url for the Cesium tile assets. They no longer allow public access anymore so you get that 403 response. In production that would be changed to point to a real tile server, but for now it’s enough just to verify that the proxy url is forwarding the request and getting the same 403 response.

Note - Unable to test this, since don't have a compatible DDF-UI with Java 11. When DDF-UI is upgraded to Java 11 can test it via instructions here https://github.com/codice/ddf/pull/6401.

###### **WFS Query:**
1. Set up a WFS server. See attachment ([geoserver.zip](https://github.com/codice/ddf/files/6240255/geoserver.zip)  There’s a readme in there that will tell you how to build it and configure the WFS source in DDF)
2. Add a WFS source to DDF pointing to the WFS server via Admin UI -> System -> WFS v1.1.0 Federated Source.
	a. Open chrome Console before clicking "Save Changes".
	b. After Clicked on "Save Changes". Ensure it save successfully and don't have any error in console and the loading icon is not infinitely loading. (Can also try same thing in new Admin UI Console)
3. Open the search UI (/search/simple will work fine) and perform queries against the wfs source.
4. Verify you get results back and that there are no errors in the logs

###### **Content Directory Monitor:**
Can follow instructions here -https://github.com/codice/ddf/pull/6401

###### **User account deletion:**
Can follow instructions here -https://github.com/codice/ddf/pull/6401

#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by abel-connexta** [comment]
build now

**Review by shaundmorris — APPROVED** [review]

**Review by zta6 — APPROVED** [review]

File: features/camel/src/main/feature/feature.xml [hunk 1]
```diff
@@ -62,6 +62,10 @@
     <bundle>mvn:org.apache.camel/camel-util-json/${camel.version}</bundle>
     <bundle>mvn:org.apache.camel/camel-support/${camel.version}</bundle>
     <bundle>mvn:org.apache.camel/camel-base/${camel.version}</bundle>
+    <bundle>mvn:org.apache.camel/camel-base-engine/${camel.version}</bundle>
+    <bundle>mvn:org.apache.camel/camel-core-processor/${camel.version}</bundle>
+    <bundle>mvn:org.apache.camel/camel-core-model/${camel.version}</bundle>
+    <bundle>mvn:org.apache.camel/camel-core-reifier/${camel.version}</bundle>
```
Comment (SmithJosh) [inline features/camel/src/main/feature/feature.xml:68]: Can't comment on the line, but the `camel-caffeine-lrucache` was removed from this feature. See https://github.com/apache/camel-karaf/compare/a387a14e1ec53344c887d479b99f5de8164adf14..7b981492e233b3cefc062bade4a9c3b6e69382e9?diff=split#diff-3fe1615c06798db8bb75db945e31236a581a944b949ae5b0066d0bc063a7b28cL62

**Review by SmithJosh — APPROVED** [review]
✏️ It'd be worth making a pass through the camel dependency versions to keep the versions in sync with what camel is using. Looks like a couple may have changed https://github.com/codice/ddf/blob/master/features/pom.xml#L50-L62

**Comment by abel-connexta** [comment]
build now

**Comment by zta6** [comment]
🎉 Hero Successful 🎉 
HttpProxy, WFS Server, Content Directory Monitor and User Account Deletion all working properly

**Pull request merged**